repos/system_upgrade/common/actors/checkyumpluginsenabled/libraries/checkyumpluginsenabled.py

```diff
@@ -0,0 +1,50 @@
+from leapp import reporting
+from leapp.libraries.common.rhsm import skip_rhsm
+
+REQUIRED_YUM_PLUGINS = ['subscription-manager', 'product-id']
+FMT_LIST_SEPARATOR = '\n    - '
+
+
+def check_required_yum_plugins_enabled(yum_config):
+    """
+    Checks whether the yum plugins required by the IPU are enabled.
+
+    If they are not enabled, a report is produced informing the user about it.
+    Does not inhibit the IPU.
+
+    :param yum_config: YumConfig
+    """
+
+    if skip_rhsm():
```

Comment:
Just a comment on this line: While currently true, that if skip_rhsm() is true we don't need to do any checks, we might want to have it remove the plugins from the required list if skip_rhsm is true instead. But right now I think this might be acceptable as is.

Reply:
Thanks for the info. I've changed in in such a way that the currently required plugins just get removed from the set of required plugins if `LEAPP_NO_RHSM` is set. 

The change was motivated by the fact that although the old implementation works, it does not really reflects the intention - when the `LEAPP_NO_RHSM` is set that means we should not use the `subscription-manager`. Since we are not supposed to be using it, we should not require its yum plugins. The old implementation suggested that when when we are not using `subscription-manager`, we do not require _any_ plugins which is wrong from the semantic point of view.

Really appreciate the comment that enabled me to see it in this fashion. Thanks!